README.md

```diff
@@ -91,15 +91,15 @@ Depending on the scenario, choose one of the two dependencies: `io.seata:seata-a
 <dependencies>
 <!--dependencies for non-SpringBoot application framework-->
   <dependency>
-    <groupId>io.seata</groupId>
+    <groupId>org.apache.seata</groupId>
```

Comment:
It is recommended to revert readme temporarily.

Reply:
done